test/Makefile.am

```diff
@@ -89,6 +89,7 @@ unittest_SOURCES = \
     ../common/Unit.cpp \
     ../common/StringVector.cpp \
     ../net/Socket.cpp \
+	../net/NetUtil.cpp \
```

Comment:
Cosmetic - indentation...

Reply:
Ironic that I had noticed all the tabs and made sure (failed, of course) to use spaces (I blame my editor). I replaced the tabs to spaces.